lint errors for api and options variables never used

https://github.com/priceline/design-system/blob/93839458b7414e22dc0b5f63dde7aabea6f05f06/codemods/icons-v2-to-v3/iconsV2toV3.js#L1

Seeing these errors when I run lint after pulling the latest:

  1:38  error  'api' is defined but never used      no-unused-vars
  1:43  error  'options' is defined but never used  no-unused-vars

![image](https://user-images.githubusercontent.com/5324905/88309200-fe9fa180-ccdb-11ea-92bd-6f42929af021.png)

--- a/codemods/icons-v2-to-v3/iconsV2toV3.js
+++ b/codemods/icons-v2-to-v3/iconsV2toV3.js
@@ -1,4 +1,4 @@
-module.exports = function (fileInfo, api, options) {
+module.exports = function (fileInfo) {
   // Sample:
   // import Dollar from 'pcln-icons/lib/Dollar'
   // import Dollar from "pcln-icons/lib/Dollar";

